src/hocon_scanner.xrl

```diff
@@ -50,11 +50,14 @@ Float               = {Integer}?{Fraction}|{Integer}{Fraction}{Exponent}
 
 %% String
 Hex                 = [0-9A-Fa-f]
-Escape              = ["\\bfnrt]
+EscapeNoQuote       = [\\bfnrt]
+Escape              = "|{EscapeNoQuote}
 UnicodeEscape       = u{Hex}{Hex}{Hex}{Hex}
 Char                = ([^\"{LineFeed}]|\\{Escape}|\\{UnicodeEscape})
 String              = "{Char}*"
-MultilineChar       = ([^"]|"[^"]|""[^"]|\\{Escape}|\\{UnicodeEscape})
+%% Special handling for trailing quote: if we don't assert it's not followed by two other
+%% quotes, `{Escape}` would "eat" one of the quotes in the triple quote...
+MultilineChar       = (\\"[^"][^"]|[^"]|"[^"]|""[^"]|\\{EscapeNoQuote}|\\{UnicodeEscape})
```

Comment:
`\\{Escap}` is changed to `\\{EscapenoQuote}`
does it mean `\\` is now parsed as `\\`, but not `\` ?

Reply:
nvm, \\ is in EscapNoQuote, it's no quote, not no backslash.


I wonder if \\ should have been added in MultilineChar at all.
it's a breaking change if we remove it though.

Reply:
not sure I understand the question.

the regex for `EscapeNoQuote` is simple the old `Escape` without `"`.

Reply:
I've just tried your suggestion, and with the fix I added to escape contents of triple quotes that are not multiline, it seems to work.

Reply:
P.S.: somehow, GH showed only your 1st comment when I wrote my 1st reply :see_no_evil: :hear_no_evil: :speak_no_evil: